bin/setup.sh

```diff
@@ -219,6 +219,14 @@ function check_i2c_sensors {
     check_device "AM2320" "5c" "10" "0.002"
 }
 
+create_bash_script_symlinks() {
+    if [[ ! -d "${HOME}/.local/bin/" ]] ; then
+        mkdir "${HOME}/.local/bin/" ;
+    fi
+    ln -fs "${INSTALL_DIR}/bin/light.sh" "${HOME}/.local/bin/light"
```

Comment:
the home local bin directory doesnt exist in $PATH environment variable by default. Lets use /usr/local/bin
ocho $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games

Reply:
It's conditional on the directory existing, creating the directory and `source`ing would indeed add it to the path. That said, `/usr/local/bin/` is also fine.